EventMixing/src/Mu2eProductMixer.cc

```diff
@@ -350,10 +360,15 @@ namespace mu2e {
                                                  CosmicLivetime& out,
                                                  art::PtrRemapper const& remap)
   {
-    if(in.size() > 1)
-      throw cet::exception("BADINPUT")<<"Mu2eProductMixer/evt: can't mix CosmicLiveTime" << std::endl;
-    for(const auto& x: in) {
-      out = *x;
+
+    if(!in.empty()) {
+      area_ = in[0]->area();
+      lowE_ = in[0]->lowE();
+      highE_ = in[0]->highE();
+      fluxConstant_ = in[0]->fluxConstant();
+      for(const auto& x: in) {
```

Comment:
Is there a use case for mixing >1 cosmic events where this calculation makes sense?  If not, it would be better to throw, as before.